modules/custom/az_core/config/install/smart_title.settings.yml

```diff
@@ -1,9 +1,8 @@
 smart_title:
-  - 'node:az_carousel_item'
-  - 'node:az_course'
   - 'node:az_event'
-  - 'node:az_news'
   - 'node:az_flexible_page'
+  - 'node:az_news'
   - 'node:az_person'
+  - 'taxonomy_term:az_news_tags'
   - 'taxonomy_term:az_person_categories'
   - 'taxonomy_term:az_person_categories_secondary'
```

Comment:
We should keep the settings for 
```
- 'node:az_carousel_item'
- 'node:az_course'
```
try `git checkout main modules/custom/az_core/config/install/smart_title.settings.yml`